Update Readme for Component class inside Sidecar folder (#443)
github/view_component: README.md

      create  app/components/example_component/example_component.html.erb
```

> #### Component inside Sidecar directory

Reviewer: ```suggestion
#### Component file inside Sidecar directory
```

> ViewComponents also support the possibility of placing the Ruby component class inside the sidecar directory, grouping all related files in the same folder.

Reviewer: ```suggestion
It's also possible to place the Ruby component file inside the sidecar directory, grouping all related files in the same folder:
```

> > **Avoid** giving your containing folder the same name as your `.rb` file or there will be a conflict between Module and Class definitions

Reviewer: ```suggestion
_Note: Avoid giving your containing folder the same name as your `.rb` file or there will be a conflict between Module and Class definitions_
```

```
app/components
├── ...
├── example
|   ├── component.rb
|   ├── component.css
|   ├── component.html.erb
|   └── component.js
├── ...

```

> Rendering is as simple as using the folder's name as a namespace:

Reviewer: ```suggestion
The component can then be rendered using the folder name as a namespace:
```
